docs/GETTING_STARTED.md

```diff
@@ -1,288 +1,247 @@
-# Getting Started With SOCI Snapshotter
-This document is a guide to walk through an example of using the
-soci-snapshotter.  To get everything running we will need to do a few things:
-- prerequisites
-- start a local references-enabled OCI registry
-- push an image to our registry using ctr
-- create and push a SOCI index to our registry
-- run a lazily loaded container from the registry
-
-Along the way we will make sure to inspect each of our actions to ensure that
-they worked.  This is valuable to help learn about the tool and debugging.
+# Getting Started With soci-snapshotter
+
+This document walks through how to use soci-snapshotter, including building SOCI
+index, pushing/pulling an image and associated SOCI index, and running a container
+with soci-snapshotter.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Prerequisites](#prerequisites)
+- [Install soci-snapshotter](#install-soci-snapshotter)
+- [Push an image to your registry](#push-an-image-to-your-registry)
+- [Create and push SOCI index](#create-and-push-soci-index)
+  - [Create SOCI index](#create-soci-index)
+  - [(Optional) Inspect SOCI index and ztoc](#optional-inspect-soci-index-and-ztoc)
+  - [Push SOCI index to registry](#push-soci-index-to-registry)
+- [Run container with soci-snapshotter](#run-container-with-soci-snapshotter)
+  - [Configure containerd](#configure-containerd)
+  - [Start soci-snapshotter](#start-soci-snapshotter)
+  - [Lazily pull image](#lazily-pull-image)
+  - [Run container](#run-container)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 ## Prerequisites
-**docker** - to confirm that you have docker working please check with `docker
-version` and `docker images` - both should succeed.
 
-**containerd** - to confirm that you have containerd working please check with
-`ctr version` and `ctr i ls`.  The version needs to be 1.4 or greater
+- **[go](https://go.dev/doc/install)** - the minimum supported go version is 1.18.
+- **[containerd](https://github.com/containerd/containerd/blob/main/docs/getting-started.md) >= 1.4** - to confirm that you have containerd working please check with
+`sudo ctr version`.
+- **[ctr](https://github.com/containerd/containerd/blob/main/docs/getting-started.md)/[nerdctl](https://github.com/containerd/nerdctl#install)** - you need one of the containerd clients to interact with containerd/registry.
+- **fuse** - used for mounting without root access (`sudo yum install fuse`).
```

Comment:
should we have a way for them to confirm if thsese exist?

Reply:
added confirmation for go. the rest either have check or the command is idempotent